src/indicators/spinner.js

```diff
@@ -26,12 +27,15 @@ import { filterInvalidDOMProps } from '../utils'
  *
  */
 
-const propTypes = {}
+const propTypes = {
+  className: PropTypes.string
```

Comment:
When a prop isn't required, it's best to declare your intent with a default prop. For this use case, an empty string would suffice.